lib/board.ex

```diff
@@ -5,37 +5,27 @@ defmodule Board do
     %Board{board: empty_board()}
   end
 
+  def get_tile(board, {row_index, col_index}) do
+    board
+    |> Enum.at(row_index)
+    |> Enum.at(col_index)
+  end
+
+  def select_tile(board, {row_index, col_index}) do
+    List.update_at(board, row_index, fn row ->
+      List.replace_at(row, col_index, :selected)
+    end)
+  end
+
   defp empty_board do
-    %{
-      0 => empty_row(),
-      1 => empty_row(),
-      2 => empty_row(),
-      3 => empty_row(),
-      4 => empty_row(),
-      5 => empty_row(),
-      6 => empty_row(),
-      7 => empty_row(),
-      8 => empty_row(),
-      9 => empty_row()
-    }
+    List.duplicate(empty_row(), 10)
```

Comment:
Awesome! 👍

Reply:
+1 - Nice job refactoring this!

Reply:
What does `10` represent in these functions?

Reply:
@klkelley The `10` represents the number of rows that a board will have. In my next PR I've given labels to these magic numbers, so that should be clearer :)